lib/package/base/jump_to_request/jump_to_api.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:hng/app/app.locator.dart';
+import 'package:hng/models/channels_search_model.dart';
+import 'package:hng/models/user_search_model.dart';
+import 'package:hng/package/base/server-request/api/http_api.dart';
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/storage_keys.dart';
+
+class JumpToApi {
+  final _channelsApi = HttpApiService("https://channels.zuri.chat/");
+  final _dmApi = HttpApiService("https://api.zuri.chat/");
+  static final storageService = locator<SharedPreferenceLocalStorage>();
+  static String? currentUserId =
+      storageService.getString(StorageKeys.currentUserId);
+  final String allChannelsPath = '/api/v1/6145ad38285e4a18402073ad/channels/';
+  final String joinedChannelsPath =
+      '/api/v1/6145ad38285e4a18402073ad/channels/users/$currentUserId/';
+
+  Map<String, dynamic> headers = {
+    'Authorization':
+        'Bearer ${storageService.getString(StorageKeys.currentSessionToken)}'
+  };
+
+  /// Fetches a list of  all channels in that organization
+  Future<List<ChannelsSearch>> allChannelsList() async {
+    print(storageService.getString(StorageKeys.currentSessionToken));
```

Comment:
Remove the print statement and make use of stacked_logger

`log.i()`

Reply:
Changes made